assets/css/book-card.css

```diff
@@ -15,7 +15,7 @@
 }
 
 .book-cards .book-card .book-info h2 {
-  @apply text-2xl text-left font-bold normal-case mt-0 lg:mt-4 before:hidden;
+  @apply text-2xl text-left font-bold normal-case mt-0 before:hidden;
```

Comment:
Remove unneeded padding from the top of the book info div at the large screen breakpoint.